src/components/CPU/RegPanel.css

```diff
@@ -1,18 +1,19 @@
 #RegPanel {
 	display: flex;
-	flex: 1 1 auto;
+	flex: 1;
 	flex-direction: column;
 	width: 22ex;
 }
 
 #RegPanel dl {
+	flex: 1 1 auto;
 	font-family: Consolas, monospace;
 	line-height: 0.8em;
 	margin: 0;
 	margin-top: 1px;
-	height: calc(28em + 71px);
 	overflow-y: auto;
 	text-align: start;
+	height: 0;
```

Comment:
Hm, I think it'd make more sense to set the basis to 0, rather than height (that is, `flex: 1 1;`.)  Would also prefer to keep properties alphabetically sorted.

-[Unknown]

Reply:
No problem, I'll fix sorting.

I spent quite a while trying to figure this out, just setting flex is not enough. I was trying to follow this SO [answer](https://stackoverflow.com/questions/41674979/flex-child-is-growing-out-of-parent) and make each item in the hierarchy fill the parent flex container without messing with `height` and finally for the `RegList` do scrolling. However, the scrollbar won't show if the element doesn't have size set so I tried a few things. I got it working with `min-height: 0` which I guess makes more sense here but that didn't work on IE 11 - (I think I hit [flexbug 3](https://github.com/philipwalton/flexbugs#flexbug-3)). I don't know if you even want to still support IE 11 but `height: 0` works. Maybe there's a better solution, I'm not very experienced with frontend dev.

Reply:
Ah, well, I had IE11 working initially so I figured I'd keep it as long as it is reasonable effort.  That said, I think we can abandon it if it has any crazy bugs - most people would be willing to open a different browser to use the debugger, is my guess.

-[Unknown]